### PR TITLE
Toolbar

### PR DESCRIPTION
**Pull request by Nirad**

### CODE REVIEW

**Comment by Nirad** [comment]
I think I have forgot clang-format ...

File: src/Gumps/GumpToolbar.cpp [hunk 1]
```diff
@@ -0,0 +1,174 @@
+//
+//  GumpToolbar.cpp
+//  crossuo
+//
+//  Created by Jean-Martin Miljours on 19-12-21.
+//
+
```
Comment (fungos) [inline src/Gumps/GumpToolbar.cpp]: Put this instead, we need to start getting these standardized as we create new files:
```
// AGPLv3 License
// Copyright (c) 2020 Jean-Martin Miljours
// Copyright (c) 2020 CrossUO Team
```

File: src/Gumps/GumpToolbar.h [hunk 2]
```diff
@@ -0,0 +1,35 @@
+//
+//  GumpToolbar.hpp
+//  crossuo
+//
+//  Created by Jean-Martin Miljours on 19-12-21.
+//
```
Comment (fungos) [inline src/Gumps/GumpToolbar.h]: Same copyright header as above

File: src/ScreenStages/GameScreen.cpp [hunk 3]
```diff
@@ -87,6 +88,7 @@ void CGameScreen::Init()
 
     g_ScreenEffectManager.UseSunrise();
     SmoothScreenAction = 0;
+    g_GumpManager.Add(new CGumpToolbar(300, 600));
```
Comment (fungos) [inline src/ScreenStages/GameScreen.cpp]: I think it should have an option to close/open this gump

Comment (Nirad) [reply src/ScreenStages/GameScreen.cpp]: ok cool I dont know what you what for this I a next pr I will add an option

File: src/Gumps/GumpToolbar.cpp (hunk 1 above)
Comment (Nirad) [reply src/Gumps/GumpToolbar.cpp]: done

File: src/Gumps/GumpToolbar.h (hunk 2 above)
Comment (Nirad) [reply src/Gumps/GumpToolbar.h]: done

**Pull request merged**